wagtail/models/__init__.py

```diff
@@ -4341,36 +4207,40 @@ class Meta:
         ]
 
 
-class TaskStateManager(models.Manager):
+class BaseTaskStateManager(models.Manager):
     def reviewable_by(self, user):
         tasks = Task.objects.filter(active=True)
         states = TaskState.objects.none()
         for task in tasks:
             states = states | task.specific.get_task_states_user_can_moderate(user=user)
         return states
 
+
+class TaskStateQueryset(SpecificQuerySetMixin, models.QuerySet):
```

Comment:
Nit: `TaskStateQuerySet` for consistency?

```suggestion
class TaskStateQuerySet(SpecificQuerySetMixin, models.QuerySet):
```